common/src/helpers/localize.ts

```diff
@@ -2,11 +2,38 @@ import { IntlShape } from "react-intl";
 
 export type Locales = "en" | "fr";
 
+export function isLocale(locale: string): locale is Locales {
+  return locale === "en" || locale === "fr";
+}
+
 export function getLocale(intl: IntlShape): Locales {
   const { locale } = intl;
-  if (locale === "en" || locale === "fr") {
+  if (isLocale(locale)) {
     return locale;
   }
   console.warn("Unknown locale. Defaulting to en.");
```

Comment:
I know this line is not part of your PR but the warning is making noise in the new tests that are part of it.  Would you consider removing it?
![image](https://user-images.githubusercontent.com/8978655/150149777-436e14cb-0f2e-4a25-bde4-790a26bf5367.png)

Reply:
Yeah, I can remove that. The point of this function is basically to hid any potential error from an unknown locale, so I suppose there's no reason to be noisy about it.